### DISCUSSION TITLE
Nerd font symbols are not properly rendered if not followed by a single space

### DISCUSSION THREAD

**Discussion by MrConnorKenway** [ghostty-org/ghostty#4029] [Feature Requests, Ideas] [answered]
When using some nerd font symbols such as '󰔶', '', '', it seems that the rendering will be influenced by their nearby characters:
<img width="600" alt="image" src="https://github.com/user-attachments/assets/809b5e4e-81b5-48b5-80bd-115cbc9c3886" />
<img width="500" alt="image" src="https://github.com/user-attachments/assets/d560f0a8-10b8-4fbd-932d-7d479c65ce3a" />

OS: macOS 15.1.1 (24B91)
Config: default config

**Comment by mitchellh** [comment, marked as answer]
This is done on purpose and mirrors Kitty. https://github.com/ghostty-org/ghostty/pull/1110

**Reply by MrConnorKenway** [reply]
I see. Is there any option to enlarge this scale or anti-aliasing it? The shape is a little bit distorted on my monitor.

**Reply by MrConnorKenway** [reply]
> This is done on purpose and mirrors Kitty.

Here is the same content displayed using kitty, and it seems that kitty does not scale the symbols:
<img width="600" alt="image" src="https://github.com/user-attachments/assets/2e5ac793-4667-4026-9644-6876c9b1b64b" />
Compared with ghostty using same font (with `font-thicken = true`):
<img width="600" alt="image" src="https://github.com/user-attachments/assets/533e537f-72c9-402f-b80c-c8f00c951d54" />
Can we change ghostty so that it only scales the part of PUA symbols that may introduce overlap issue?
